### PR TITLE
Migration to enable dashboard everywhere #3065

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones @lbosque please review

### CODE REVIEW

**Comment by Kartones** [comment]
I'd edit the NEWS.md file to reflect this, but otherwise :+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3467/))

File: db/migrate/20150505103504_new_dashboard_for_everybody.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      feature_flag = FeatureFlag[name: flag_name]
+      if feature_flag
+        if feature_flag.restricted
+          feature_flag.restricted = false
+          feature_flag.save
+        end
+      else
+        feature_flag = FeatureFlag.new(name: flag_name, restricted: false)
+        feature_flag.id = FeatureFlag.order(:id).last.id + 1
+        feature_flag.save
```
Comment (lbosque) [inline db/migrate/20150505103504_new_dashboard_for_everybody.rb]: Isn't there a sequence in id column?

Comment (Kartones) [inline db/migrate/20150505103504_new_dashboard_for_everybody.rb]: no, or at least is not autogenerated

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3471/))

**Pull request merged**